### ISSUE TITLE
How to start creating support for plugins?

### ISSUE THREAD

**Issue by karlomedallo** [saiichihashimoto/sanity-typed-schema-builder#187]
Hi, I'm interested in using this in our greenfield project. However, I just need guidance on how to start creating for some plugins types that we will use. An example is the color-input plugin (https://www.sanity.io/plugins/color-input). Any pointers is appreciated

**Comment by saiichihashimoto** [comment]
I haven't considered plugins at all with this, so there's a good chance they might not work. It's a good question. There's availability for creating your own types, but that's not exactly what this is.

**Comment by karlomedallo** [comment]
Thanks for the response. Can this be converted to a feature request instead?

**Comment by saiichihashimoto** [comment]
Yup! I'm not doing anything special in this repo to distinguish questions from feature requests, but that's how I'm approaching this.

**Comment by karlomedallo** [comment]
Much appreciated! I really like the idea of just writing once, and you get both Typescript types and Sanity Schema. Thank you for making this project!
